### PR TITLE
Added email-autocomplete-input and minesweeper

### PR DESCRIPTION
**Pull request by alex-cory**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,9 @@ Your contributions and suggestions are heartily welcome. =^.^=
 - [Google Play Music Desktop Remote](https://github.com/GPMDP/google-play-music-desktop-remote) A React-Native app for remote controlling Google Play Music Desktop: MobX + WebSocket.
 - [PokemonGo webspoof](https://github.com/iam4x/pokemongo-webspoof/) Play PokemonGo on your Mac
 - [vcash-electron](https://github.com/whphhg/vcash-electron) - Electron UI for the Vcash crypto currency
-- [Streamflow](https://github.com/hawkins/streamflow) - Electron app for improving the Twitch viewer experience
+- [Streamflow](https://github.com/hawkins/streamflow) - Electron app for improving the Twitch viewer experience]
```
Comment (hawkins) [inline README.md]: Looks like there's an accidental `]` at the end of this line

Comment (hawkins) [inline README.md]: Looks like there's an accidental `]` at the end of this line

Comment (alex-cory) [reply README.md]: Whoops! Yep! Will fix

**Comment by hawkins** [comment]
Sorry I didn't get around to merging this forever ago, but I'm closing this now as it's a bit outdated. Sorry! 😬

**Pull request closed without merging**